Makefile

```
@@ -748,6 +748,19 @@ testcuda test-cuda: $(ACTIVATE_VENV)
 test-workflow: test-assets core $(BIN)/ocrd $(ACTIVATE_VENV)
 	. $(ACTIVATE_VENV) && cd core/tests/assets/SBB0000F29300010000/data/ && bash -x $(CURDIR)/test-workflow.sh
 
+DOCKER_COMPOSE = docker compose
+INTEGRATION_TEST_IN_DOCKER = docker exec core_test
+
+integration-test: test-assets
+	$(DOCKER_COMPOSE) --file core/tests/network/docker-compose.yml --file tests/network/docker-compose.yml up -d
+	docker cp core/tests/assets/kant_aufklaerung_1784/data/. ocrd_network_processing_server:/data
+	# Queues must exists and it takes time until they are created by the workers. We might need
+	# a mechanism to test if all queues are there but that is not available yet. So sleeping
+	# here for now
+	sleep 10
+	-$(INTEGRATION_TEST_IN_DOCKER) pytest -k 'test_ocrd_all_workflow' -v
```

```suggestion
	$(INTEGRATION_TEST_IN_DOCKER) pytest -k 'test_ocrd_all_workflow' -v
```

We want failure of the test command in failure of the `make integration-test` target, no?

---

On the CI/CD pipeline, yes. However, when testing locally, failed tests leave leftovers. Potentially, we need both versions?

---

But for that use-case we used to have the convention of passing a variable `PYTEST_ARGS` where the user could insert things like `--continue-on-collection-errors`, right?